src/Maestro/tests/DependencyUpdateErrorProcessor.Tests/UpdateGithubIssueTest.cs

```diff
@@ -0,0 +1,131 @@
+            Context.Subscriptions.Add(subscription.Object);
+            Context.SaveChanges();
+
+            Mock<Octokit.Repository> repository = new Mock<Repository>();
+            GithubClient.Setup(x => x.Repository.Get(It.IsAny<string>(), It.IsAny<string>())).ReturnsAsync(repository.Object);
+            Issue issueCreated = GetIssue();
+            List<NewIssue> newIssues = new List<NewIssue>();
+            GithubClient.Setup(x => x.Issue.Create(It.IsAny<long>(), Capture.In(newIssues))).ReturnsAsync(issueCreated);
+            GithubClient.Setup(x => x.Issue.Get(0, 1)).ReturnsAsync(issueCreated);
+            List<IssueUpdate> issueUpdate = new List<IssueUpdate>();
+            GithubClient.Setup(x => x.Issue.Update(0, 1, Capture.In(issueUpdate))).ReturnsAsync(issueCreated);
+            DependencyUpdateErrorProcessor errorProcessor =
+                ActivatorUtilities.CreateInstance<DependencyUpdateErrorProcessor>(Scope.ServiceProvider,
+                    Context);
+            await errorProcessor.ProcessDependencyUpdateErrorsAsync();
+            string issueBody = $@"The following errors have been detected when attempting to update dependencies in 
+'https://github.com/maestro-auth-test/maestro-test2'
+
+ [marker]: <> (subscriptionId: 'ee8cdcfb-ee51-4bf3-55d3-08d79538f94d', method: 'UpdateAssetsAsync', errorMessage: 'build: 14	Unexpected error processing action: Validation Failed')
+**SubscriptionId:** 'ee8cdcfb-ee51-4bf3-55d3-08d79538f94d'
+**Source Repository :**  'Source Repo'
+**Target Repository :**  'Target Repo'
+**Branch Name :**  '38'
+**Error Message :**  'build: 14	Unexpected error processing action: Validation Failed'
+**Method :** 'UpdateAssetsAsync'
+**Action :** 'Creating new issue'
+**Last seen :** '1/1/2200 12:00:00 AM'
+**/FyiHandle :** @epananth";
+            string updatedIssueBody =
+                $@"The following errors have been detected when attempting to update dependencies in
+'https://github.com/maestro-auth-test/maestro-test2'
+
+  [marker]: <> (subscriptionId: 'ee8cdcfb-ee51-4bf3-55d3-08d79538f94d', method: 'UpdateAssetsAsync', errorMessage: 'build: 14	Unexpected error processing action: Validation Failed')
+**SubscriptionId:** 'ee8cdcfb-ee51-4bf3-55d3-08d79538f94d'
+**Source Repository :**  'Source Repo'
+**Target Repository :**  'Target Repo'
+**Branch Name :**  '38'
+**Error Message :**  'build: 14	Unexpected error processing action: Validation Failed'
+**Method :** 'UpdateAssetsAsync'
+**Action :** 'Updating existing issue'
+**Last seen :** '2/1/2200 12:00:00 AM'
+**/FyiHandle :** @epananth";
+            GithubClient.Verify(x => x.Issue.Update(0, 1, It.IsAny<IssueUpdate>()), Times.Once);
+            Assert.Single(newIssues);
+            Assert.Equal("[Dependency Update] Errors during dependency updates to : https://github.com/maestro-auth-test/maestro-test2", newIssues[0].Title);
+            Assert.Equal("DependencyUpdateError", newIssues[0].Labels[0]);
```

Comment:
I think there is an Assert.Contains, which would make more sense... don't assert the first one is this value... Assert.Single, and then Assert.Contains()... that way it will be clearer when it fails as to why it failed (either there are too many, or it doesn't contain what we are looking for)